src/components/Topnav.vue

```diff
@@ -101,6 +101,7 @@ onMounted(() => setTitle());
               v-if="!$auth.isAuthenticated.value"
               @click="modalAccountOpen = true"
               :loading="loading || web3.authLoading"
+              aria-label="Login"
```

Comment:
It should be connect wallet with translation
```suggestion
              :aria-label="$t('connectWallet')"
```